main.cpp

```diff
@@ -137,10 +132,7 @@ bool Skateboard::kickFlip(int degree)
         std::cout << "rad!\n.";
         return true;
     }
-    else
-    {
         return false;
-    }
 }
```

Comment:
Just because you delete the `else { }` doesn't mean you can ignore the formatting and not fix the indentation.
- [ ] revise